src/reducers/spendings.js

```diff
@@ -0,0 +1,36 @@
+import {confirm} from '../functions';
+
+
+export function spendingsReducer(state = [], action) {
+    switch (action.type) {
+
+        case "ADD_SPENDINGS":
+            let newSpendings = {
+                    title: action.text
+                };
+                return [
+                ...state,
+                newSpendings
+            ];
+
+        case "REMOVE_SPENDINGS":
+            if (confirm(action.title)) {
+                let newArr = state.filter(item => item.title !== action.title);
+                    return [...newArr];
+            }
+            return state;
+
+        case "ADD_USERS_TO_SPENDS":
```

Comment:
I'd rather introduced some universal action like  `UPDATE_SPENDINGS` and identified spending with a spending `title`, like you already do to remove it.

the interface should contain two arguments
`action.users`
`action.title`

you will be able to find the record with
```javascript
state.find(item => item.title === action.title);
```

with such action you will be able to update any spending with extra data, instead of just the last one as in the code right now